cmd/mbop/main.go

```diff
@@ -186,23 +215,35 @@ func getUser(w http.ResponseWriter, r *http.Request) (*User, error) {
 		return &User{}, fmt.Errorf("can't create keycloak client: %s", err.Error())
 	}
 
-	_, err = k.GetGenericToken("redhat-external", username, password)
+	oauthClientConfig := clientcredentials.Config{
+		ClientID:       "admin-cli",
+		ClientSecret:   "",
+		TokenURL:       KEYCLOAK_SERVER + "auth/realms/redhat-external/protocol/openid-connect/token",
```

Comment:
this requires a `/`  , the `KEYCLOAK_SERVER` is not expected to have a trailing `/' . 

```suggestion
		TokenURL:       KEYCLOAK_SERVER + "/auth/realms/redhat-external/protocol/openid-connect/token",
```